_includes/sections/working-here/your-pay-pension-and-other-benefits.md

```diff
@@ -108,38 +108,100 @@ your holiday allowance shown on your BreatheHR dashboard.
+work. Your line manager will then review and close the sick leave. If you're
+sick for more than 7 calendar days in a row, you'll also need to provide a fit
+note.
+
+##### Absence meetings
+
+Your line manager should check-in with you on a regular basis about your
+wellbeing, and catch up with you after any sickness absence to see how you are.
+If you're sick for more than 7 working days over a rolling 12 month period, your
+line manager will invite you to an absence meeting to discuss your health. Where
+appropriate, we’ll talk about how we can support your return to work and any
+temporary or permanent adjustments which might help improve your wellbeing and
+reduce your levels of absence. We might also agree an action plan and / or a
+review period. Where we think it would be helpful, we’ll seek medical advice via
+an Occupational Health advisor.
+
+If you have a long term health condition and would like to discuss reasonable
+adjustments and ways of working that will help, we welcome a conversation at any
+point. The sooner we know, the sooner we can support you.
+
+##### Pay
+
+Within the 12 month period, a person will receive:
+
+- full pay for the first 12 days of sickness absence
+- half pay (but no less than living wage) for the next 8 days of sickness
+  absence
+- living wage for the next 10 days of sickness absence
+- statutory sick pay pay after 30 days of sickness absence
+
+There'll be circumstances where we'll make exceptions, because it's not possible
+to write a policy that is fair in every situation. This structure will form our
+starting point for those decisions. All decisions on exceptions will be made by
+the Directors’ group. Our aim is to ensure that we’re treating everyone fairly
+and consistently.
+
+##### Long term medical treatment
+
+Regular and open communication with your line manager is important to ensuring
+we can support you in the best possible way through a period of long term
+medical treatment (a period of more than 2 weeks).
+
+If you require long term medical treatment that would affect your ability to
+work as normal, but you are still able to work, we will support this wherever we
+can. We may consider a change of project or role where that’s an option.
+
+If you finish your treatment, you’ll have a meeting to aid your return to work,
+or to normal working hours, and ensure your ongoing wellbeing is supported. We
+may look at a phased return for example.
+
+##### Escalation
+
+If your sickness absence continues for 3 months after an absence meeting, your
+line manager will set up a further meeting to discuss your situation. We will
+look to make any further reasonable adjustments to help you get back working.
+This might include looking for a suitable alternative role that fits your
+circumstances better.
+
+Where we’re unable to reach an agreement that means you’re able to maintain a
+reasonable level of attendance for work, we may consider your dismissal. We will
```

Comment:
It would be good to be clear about what a reasonable level of attendance is, whether it's defined for the whole company, role specific, or whether it's something to be negotiated if such a situation comes up.

Reply:
It would be negotiated and discussed depending on the circumstance. We can't predict for these situations.

Reply:
Can we right down how we'd determine what a resonable level is?

```suggestion
reasonable level of attendance for work, as determined by you, your line manager, and the HR team, we may consider your dismissal. We will
```